lib/mpl_toolkits/axisartist/floating_axes.py

```diff
@@ -223,24 +220,23 @@ def _update_grid(self, x1, y1, x2, y2):
         lon_lines, lat_lines = grid_finder._get_raw_grid_lines(
             lon_values[(lon_min < lon_values) & (lon_values < lon_max)],
             lat_values[(lat_min < lat_values) & (lat_values < lat_max)],
-            lon_min, lon_max, lat_min, lat_max)
+            tbbox)
 
         grid_info["lon_lines"] = lon_lines
         grid_info["lat_lines"] = lat_lines
 
         lon_lines, lat_lines = grid_finder._get_raw_grid_lines(
-            # lon_min, lon_max, lat_min, lat_max)
-            extremes[:2], extremes[2:], *extremes)
+            tbbox.intervalx, tbbox.intervaly, tbbox)
 
         grid_info["lon_lines0"] = lon_lines
         grid_info["lat_lines0"] = lat_lines
 
     def get_gridlines(self, which="major", axis="both"):
         grid_lines = []
         if axis in ["both", "x"]:
-            grid_lines.extend(self._grid_info["lon_lines"])
+            grid_lines.extend([xys.T for xys in self._grid_info["lon_lines"]])
```

Comment:
No need for intermediate list here (and elsewhere)? (But not sure that I follow the underlying format change, so may still be required?)

Reply:
Actually the list comprehension is faster, so if one cares about performance one should add the extra surrounding brackets here:
```
In [4]: %timeit l = range(100); m = []; m.extend([x ** 2 for x in l])
3.2 µs ± 14.5 ns per loop (mean ± std. dev. of 7 runs, 100,000 loops each)

In [5]: %timeit l = range(100); m = []; m.extend(x ** 2 for x in l)
4.71 µs ± 182 ns per loop (mean ± std. dev. of 7 runs, 100,000 loops each)
```
I'm almost certain I already made that comment on another thread, but I can't find it.
(This may change in the future if genexps get inlined, per the comment at the end of https://peps.python.org/pep-0709/#specification)